src/lib/components/phoneInput/phoneInput.js

```diff
@@ -54,21 +56,24 @@ class PhoneInput extends PureComponent {
       onCountrySelected,
       ...inputProps
     } = this.props;
+    // eslint-disable-next-line no-console
+    console.log({ theme, style: styles[`theme-${theme}`], styles });
```

Comment:
Hey @kbardi, I don't understand why we need this here

Reply:
That's true! we don't need it